apps/api/src/app/app.module.ts

```diff
@@ -179,11 +183,8 @@ const autoSchemaFile = environment.production
     ApiCatalogueModule,
     IdentityModule,
     AuthModule.register(environment.auth),
-    SyslumennModule.register({
-      url: environment.syslumennService.url,
-      username: environment.syslumennService.username,
-      password: environment.syslumennService.password,
-    }),
+    SyslumennClientModule,
```

Comment:
Should load SyslumennClientConfig here, but only import it where needed (inside SyslumennModule).